parts/printers/Prusa_MK3S.h

```diff
@@ -66,7 +66,7 @@ class Prusa_MK3S : public Boards::EinsyRambo, public Printer
 	private:
 		void FixSerial(avr_t * avr, avr_io_addr_t addr, uint8_t v);
 
-		std::atomic_uchar m_key = {0}, m_mouseBtn = {0};
+		std::atomic<int> m_key = {0}, m_mouseBtn = {0};
```

Comment:
I think we do want to keep the atomic_[something] type if possible. IIRC the specialized versions take advantage of faster locking options whereas the templated verison has the cost of a standard mutex. (But it's been awhile since I looked at the docs on that)

Reply:
yeah, I can flip this back

Reply:
Perfect, thanks!